spring-cloud-vault-config-databases/pom.xml

```diff
@@ -95,6 +95,11 @@
 			<artifactId>spring-boot-starter-jdbc</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+		  <groupId>com.couchbase.client</groupId>
+		  <artifactId>java-client</artifactId>
+		  <version>3.0.5</version>
+		</dependency>
```

Comment:
This dependency goes into `test` scope. The version should be inherited from Spring Boot.

Reply:
Not quite sure what you meant but I changed the dependency to
````xml
	<dependency>
	  <groupId>org.springframework.data</groupId>
	  <artifactId>spring-data-couchbase</artifactId>
	</dependency>
````

Reply:
The dependency as it's defined here is a hard dependency on Spring Data Couchbase. Anyone depending on `spring-cloud-vault-config-database` would pull Spring Data Couchbase into their project regardless of whether they intend to use Couchbase or not. Please add `<scope>test</scope>` to avoid a hard dependency.